__test__/backend.test.js

```diff
@@ -740,10 +740,10 @@ describe.only("editUser mutation", () => {
 
   it("returns the user if it is called with a userId by no userData", async () => {
     const result = await runGql(editUserMutation, variables, testAdminUser);
-    expect(result).toEqual({
+    expect(result).toMatchObject({
       data: {
         editUser: {
-          id: "2",
+          // id: "2", // id might be diff
```

Comment:
Shouldn't we just delete this line then?

Reply:
I thought it would be good to document why we are not comparing by equality (and in case there are other tests, a place where it's shown how to fix)